report-handling-procedure.md

```diff
@@ -0,0 +1,197 @@
+:point_left: [Come back to index](README.md)
+
+# Chicago Python Users Group Code of Conduct
+
+The following guide was borrowed almost-entirely from the excellent Django Code
+of Conduct [Report Handling
+Procedure](https://github.com/django/code-of-conduct/blob/master/reports.md).
+
+## Report handling procedure
+
+This document aims to present the way we handle received reports. This is not a
```

Comment:
nit: *We* - could the On-call duty section be moved up before this section? This is the opening sentence and it is not clarified who the we represents. Moving the section would be helpful for the reader.

Reply:
Good call. This also helped me address the attribution problem.